tests/distributed/test_multi_node_topology.py

```diff
@@ -0,0 +1,75 @@
+"""Compare the outputs of HF and distributed vLLM when using greedy sampling.
+
+Run:
+```sh
+cd $VLLM_PATH/tests
+
+pytest distributed/test_multi_node.py
+```
+"""
+import os
+
+import pytest
+import ray
+from ray.cluster_utils import Cluster
+
+from vllm.utils import cuda_device_count_stateless
+
+TARGET_TEST_SUITE = os.environ.get("TARGET_TEST_SUITE", "L4")
```

Comment:
please clean up the tests and remove unnecessary code. this test does not need `TARGET_TEST_SUITE` I think.